### PR TITLE
pythonPackages.seqnmf: init at 0.1.2

### PR DESCRIPTION
**Pull request by tbenst**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/seqnmf/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  propagatedBuildInputs = [
+    numpy
+    matplotlib
+    scipy
+    seaborn
+  ];
+
+  # no tests
+  doCheck=false;
```
Comment (jonringer) [inline pkgs/development/python-modules/seqnmf/default.nix]: looking at https://github.com/ejolly/seqnmf, it looks like they don't have any tests, and haven't had an update in 18+ months :(

Comment (jonringer) [reply pkgs/development/python-modules/seqnmf/default.nix:27]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

**Comment by onny** [comment]
Please rebase

**Comment by AndersonTorres** [comment]
Closing as dead

**Pull request closed without merging**